### PR TITLE
Klog flags

### PR DESCRIPTION
**Pull request by vrutkovs**
This should resolve `flag provided but not defined: -logtostderr` error when clusterapi start machine controller

### CODE REVIEW

**Comment by russellb** [comment]
This code was based on the docs from https://github.com/kubernetes-sigs/cluster-api/tree/master/docs/book/provider_implementations

You may want to consider proposing this addition there, too.

**Comment by vrutkovs** [comment]
Filed https://github.com/kubernetes-sigs/cluster-api-provider-openstack/issues/271 to track this. I'm not entirely sure if this PR would fix it, so didn't propose upstream PR yet

**Comment by russellb** [comment]
I tried this fix, but it doesn't seem to work for me.

```
bin/machine-controller-manager --logtostderr=true --v=3
flag provided but not defined: -logtostderr
```

File: cmd/manager/main.go [hunk 1]
```diff
@@ -33,18 +34,30 @@ import (
 )
 
 func main() {
```
Comment (russellb) [inline cmd/manager/main.go]: This patch works if I add this to the beginning of main():

```
    klog.InitFlags(nil)
```

Comment (vrutkovs) [reply cmd/manager/main.go]: Awesome, thanks!

**Pull request merged**